Add table for projections (#753)
GenericMappingTools/pygmt: examples/projections/table/README.txt

Projection Table
----------------

> PyGMT offers 31 map projections. These are specified using the ``projection`` argument.
> The projection codes for the projection styles are shown below.

Reviewer: There are similar sentences at the top of this page (https://pygmt-f6khtwuqs.vercel.app/projections/index.html).
```suggestion
The below table shows the projection codes for the 31 GMT projections.
```

Reviewer: @willschlitzer I'm not a native English speaker, you can rewrite the sentence if it doesn't read well.

Author: Looks good to me!

.. |lonp| replace:: lon\ :sub:`p`
.. |latp| replace:: lat\ :sub:`p`

.. list-table::
   :widths: 20 28
   :header-rows: 1

   * - PyGMT Projection Argument
     - Projection Name
   * - **A**\ |lon0|/|lat0|\ [/\ *horizon*]/\ *width*
     - Lambert azimuthal equal area
   * - **B**\ |lon0|/|lat0|/|lat1|/|lat2|/\ *width*
     - Albers conic equal area
   * - **C**\ |lon0|/|lat0|/\ *width*
     - Cassini cylindrical
   * - **Cyl_stere/**\ [|lon0|\ [/|lat0|/]]\ *width*
     - Cylindrical stereographic
   * - **D**\ |lon0|/|lat0|/|lat1|/|lat2|/\ *width*
     - Equidistant conic
   * - **E**\ |lon0|/|lat0|\ [/\ *horizon*]/\ *width*
     - Azimuthal equidistant
   * - **F**\ |lon0|/|lat0|\ [/\ *horizon*]/\ *width*
     - Azimuthal gnomonic
   * - **G**\ |lon0|/|lat0|\ [/\ *horizon*]/\ *width*
     - Azimuthal orthographic
   * - **G**\ |lon0|/|lat0|/\ *alt*/*azim*/*tilt*/*twist*/*W*/*H*/*width*
     - General perspective
   * - **H**\ [|lon0|/]\ *width*
     - Hammer equal area
   * - **I**\ [|lon0|/]\ *width*
     - Sinusoidal equal area
   * - **J**\ [|lon0|/]\ *width*
     - Miller cylindrical
   * - **Kf**\ [|lon0|/]\ *width*
     - Eckert IV equal area

>    * - **-Ks**\ [|lon0|/]\ *width*

Reviewer: ```suggestion
   * - **Ks**\ [|lon0|/]\ *width*
```

Projection Table
----------------

PyGMT offers 31 map projections. These are specified using the ``projection`` argument.
The projection codes for the projection styles are shown below. Upper case arguments

> set *map width* and lower case arguments set *map scale*.

Reviewer: > Upper case arguments set *map width* and lower case arguments set *map scale*.

New users will be confused about what "Upper case" and "Lower case" mean. Please try to rephrase it, or just give an example.

Reviewer: https://pygmt-git-fork-willschlitzer-proj-table.gmt.vercel.app/projections/index.html

I just realized that we already have similar sentences at the top of this page.

> PyGMT support many map projections. Use the projection argument to specify which one you want to use in all plotting modules. The projection is specified by a one letter code along with (sometimes optional) reference longitude and latitude and the width of the map (for example, Alon0/lat0[/horizon]/width). The map height is determined based on the region and projection.

Perhaps no need to repeat it in the `projections/table/README.txt` file

Author: I removed it from the table README
